frontend/vue/components/UtilityPanel/Scratchpad.vue

```diff
@@ -78,6 +79,11 @@ export default class Scratchpad extends Vue {
     this.code = code
   }
 
+  resetOutput () {
+    const codeOutput: any = this.$refs.output
```

Comment:
The `any` is the only thing that I see weird but I assume that it's because you don't know exactly what you will receive from my-binder.